src/app/ui/search/search.component.ts

```diff
@@ -1,8 +1,8 @@
 import { Component, OnInit } from '@angular/core';
 import {Database} from '../../database/database';
 import {Child} from '../../children/child';
-import {School} from '../../schools/school';
 import {Entity} from '../../entity/entity';
+import {School} from '../../schools/school';
```

Comment:
if you notice it, try to avoid committing such small, unrelated changes. The less changes, the easier is the review and merging :-)